### DISCUSSION TITLE
这文档写的都是什么玩意？？？

### DISCUSSION THREAD

**Discussion by mclwh** [galacean/discussion#86] [Engine] [answered]
![image](https://github.com/galacean/discussion/assets/37808584/f691fee5-0f62-4adc-8d2e-5abcfe3e6d34)
这个Texture2DArr从哪来的？？？？？？？？？？？

**Comment by cptbtptpbcptdtptp** [comment]
你好，在文档的最后有源码，因为这部分不是描述加载纹理，所以没有添加加载的逻辑
![image](https://github.com/galacean/discussion/assets/7768919/5e479bd2-0ff7-4f60-b883-0e16fe7fcfe2)
将文档拉到最后，这里有配套的源码，Texture2DArr 是加载的纹理数组，请见谅。

**Reply by cptbtptpbcptdtptp** [reply, marked as answer]
现在的 flappy bird 教程已经重构过一遍，在编辑器中就可以直接按照流程实现一遍，可以参考[像素小鸟](https://galacean.antgroup.com/engine/docs/latest/cn/getting-started-flappy-bird)
